src/next-video.tsx

```diff
@@ -111,6 +114,7 @@ export default function NextVideo(props: NextVideoProps) {
         poster=""
         style={{
           '--controls': controls === false ? 'none' : undefined,
+          aspectRatio: aspectRatio !== '16/9' ? aspectRatio : undefined,
```

Comment:
What's the reasoning behind this check? We don't want to use the aspect ratio if it's `4:3` or otherwise?

Reply:
the default is 16 / 9 defined in the inline style tag. if it's the same than 16/9 we skip setting it as an inline style